arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -1261,17 +1267,24 @@ void RocksDBEngine::trackRevisionTreeResurrection() noexcept {
 
 void RocksDBEngine::trackRevisionTreeMemoryIncrease(
     std::uint64_t value) noexcept {
-  if (value != 0) {
```

Comment:
Do you by chance remember why this was here?

Reply:
It was there only to save an atomic write in case it wouldn't be necessary.